src/java.base/share/classes/java/time/format/DecimalStyle.java

```diff
@@ -115,7 +115,9 @@
     /**
      * Lists all the locales that are supported.
      * <p>
-     * The locale 'en_US' will always be present.
+     * At a minimum, the returned Set must contain a {@code Locale} instance equal to
```

Comment:
A nit, really. Consider applying either of these suggestions:
```suggestion
     * At a minimum, the returned {@code Set} must contain a {@code Locale} instance equal to
```
```suggestion
     * At a minimum, the returned set must contain a {@code Locale} instance equal to
```

Reply:
Thanks. Adopted the former suggestion.